pkg/database/realm.go

```diff
@@ -1597,6 +1597,7 @@ func (r *Realm) ExternalIssuerStats(db *Database) (ExternalIssuerStats, error) {
 			$1 AS realm_id,
 			d.issuer_id AS issuer_id,
 			COALESCE(s.codes_issued, 0) AS codes_issued
```

Comment:
Need comma

Reply:
which means it also needs a test... looking at that too